bridge/contracts/DutchAuction.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.11;
```

Comment:
Should update this to `0.8.16` to match the https://github.com/alicenet/alicenet/pull/199

Reply:
Done. Thanks!